### DISCUSSION TITLE
Why are colons allowed in SGR codes?

### DISCUSSION THREAD

**Discussion by walles** [kovidgoyal/kitty#6798] [Q&A]
Hello!

This change was made to accept colons in SGR sequences: e90aaa847047c4dd680d8d92ccb5ac30f25e46e7

I've looked around, but been unable find it documented anywhere that this should work.

Why did Kitty decide to support this?

I probably just haven't looked in the right place, so if you have pointers to the right place or some hits about why this was added that would be much appreciated!

  Regards /Johan

**Comment by kovidgoyal** [comment]
Because the ECMA 48 specs says they must be used.
